src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+
+  const formattedSearchQuery = searchQuery.toLocaleLowerCase().trim();
```

Comment:
It would be nice to delete unnecessary spaces also in the search query. If  I search `love    actually` I will not get results